src/test/java/seedu/address/model/ModelManagerTest.java

```diff
@@ -93,6 +97,22 @@ public void getFilteredPersonList_modifyList_throwsUnsupportedOperationException
         assertThrows(UnsupportedOperationException.class, () -> modelManager.getFilteredPersonList().remove(0));
     }
 
+    @Test
+    public void getNextSession_success() {
```

Comment:
also in case it wasn't clear, use a `NextSessionUtil` stub to override `getTimeNowAsSession` so you know exactly what time the "system" will return. 

to reply your comment in the other pr, 
1. you're testing `getNextSession` not `getTimeNowAsSession` 
2. `getTimeNowAsSession`  is a private method so it doesn't have to be tested
3. you are relying on `LocalDateTime` which isn't your class to begin with so it's not something you should test and you should assume it works
4. when you test a method the only reason you can test it in the first place is by ensuring/assuming everything else works (if not how will you know where the problem lies) and you are basically testing it in isolation with everything else

Reply:
yeap as mentioned will add more test cases! also yea probably will use the stub method too